<?xml version="1.0" ?><root>
<BetterExceptions><BEversion>v3.11</BEversion><WasBlank>False</WasBlank><TuningLoadFinished>True</TuningLoadFinished><BadObjectCC>False</BadObjectCC><Advice>Not available. More info may be in BE Report.</Advice></BetterExceptions><report><version>2</version><sessionid>daddb6c857e402cc65ad8f0f</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2024-01-21 23:15:55</createtime><buildsignature>Local.Unknown.Unknown.1.104.58.1030-1.200.000.618.Release</buildsignature><categoryid>funds.py:97</categoryid><desyncid>daddb6c857e402cc65ad8f0f</desyncid><systemconfig/><screenshot/><desyncdata>Exception while generating potential interactions for object_DJBooth_Industrial:0x07e0122d06043bab: (AttributeError: 'NoneType' object has no attribute 'family_funds')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\crafting\crafting_interactions.py", line 1174, in picker_rows_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\crafting\crafting_process.py", line 481, in recipe_test_price&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\crafting\crafting_process.py", line 443, in _get_price_error_list&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\payment\payment_source.py", line 95, in max_funds&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\funds.py", line 97, in get_funds_for_source&#13;&#10;AttributeError: 'NoneType' object has no attribute 'family_funds'rtim=0&#13;&#10;Client Session Time: 23.73&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,104,0,5617&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19043 GPU: AMD Custom GPU 0405 (RADV VANGOGH) aticfx64.dll 32767.65535.65535.65535, GUID: 00000000-0004-0000-0000-000000000000&#13; &#10;</desyncdata></report>
<report><version>2</version><sessionid>daddb6c857e402cc65ad8f0f</sessionid><type>desync</type><sku>ea.maxis.sims4_64.15.pc</sku><createtime>2024-01-21 23:16:00</createtime><buildsignature>Local.Unknown.Unknown.1.104.58.1030-1.200.000.618.Release</buildsignature><categoryid>funds.py:97</categoryid><desyncid>daddb6c857e402cc65ad8f0f</desyncid><systemconfig/><screenshot/><desyncdata>Exception while generating potential interactions for object_DJBooth_Industrial:0x07e0122d06043bab: (AttributeError: 'NoneType' object has no attribute 'family_funds')&#13;&#10;Traceback (most recent call last):&#13;&#10;&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\crafting\crafting_interactions.py", line 1174, in picker_rows_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\crafting\crafting_process.py", line 481, in recipe_test_price&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\crafting\crafting_process.py", line 443, in _get_price_error_list&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\payment\payment_source.py", line 95, in max_funds&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\sims\funds.py", line 97, in get_funds_for_source&#13;&#10;AttributeError: 'NoneType' object has no attribute 'family_funds'rtim=0&#13;&#10;Client Session Time: 23.73&#13; &#10;Num Save Errors: 0&#13; &#10;Num Load Errors: 0&#13; &#10;Current Game State: 0x96d01090&#13; &#10;LoadingScreenActiveFlag: 1&#13; &#10;Origin Version: 13,104,0,5617&#13; &#10;Modded: True&#13; &#10;SystemInfo: Windows 10 10.0.19043 GPU: AMD Custom GPU 0405 (RADV VANGOGH) aticfx64.dll 32767.65535.65535.65535, GUID: 00000000-0004-0000-0000-000000000000&#13; &#10;</desyncdata></report>
</root>